code/WorkInProgress/recycling/conveyor.dm

```diff
@@ -18,12 +18,16 @@
 	anchored = ANCHORED
 	power_usage = 0
 	layer = 2
+	mouse_drag_pointer = MOUSE_ACTIVE_POINTER
 	machine_registry_idx = MACHINES_CONVEYORS
-	var/operating = OP_OFF	// 1 if running forward, -1 if backwards, 0 if off
+	mechanics_type_override = /obj/machinery/conveyor/built
+	var/operating = CONVEYOR_STOPPED	// 1 if running forward, -1 if backwards, 0 if off
 	var/operable = TRUE	// true if can operate (no broken segments in this belt run)
```

Comment:
code docs

Reply:
Was there before I touched the code, but alright.